README.md

```diff
@@ -445,6 +446,74 @@ At this point, we can start the web application as follows:
 
 Please consult the `help` CLI command for more insights on the various options.
 
+
+# Configuration
+
+To ease life of users, Trento can be run with a config file in replacement of command-line arguments.
```

Comment:
We should mention here the ability to use environment variables automatically mapped to config keys.
I would also suggest to remove the sentence `To ease life of users,`, because that's a very subjective thing. 😛

Reply:
`To ease life of users` :trollface: 
Fixing it.